# calibrated parameters, calculated from \scout_odom
Header header
float64 apparent_baseline
float64 gear_ratio
float64 last_apparent_baseline
float64 last_gear_ratio
